srcpkgs/gdown/template

```diff
@@ -1,17 +1,19 @@
 # Template file for 'gdown'
 pkgname=gdown
-version=3.12.2
-revision=2
+version=3.13.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3 python3-setuptools"
 depends="python3-filelock python3-requests python3-six python3-tqdm
  python3-pysocks"
+checkdepends="${depends} python3-pytest"
 short_desc="Download a large file from Google Drive"
 maintainer="travankor <travankor@tuta.io>"
 license="MIT"
 homepage="https://github.com/wkentaro/gdown"
 distfiles="https://github.com/wkentaro/gdown/archive/v${version}.tar.gz"
-checksum=c2c58283236b87371ede8692626c44a839e1a962d48eceb269841535bef21a73
+checksum=830d88d3ab4640ff2f0a34d51bda4badda64ed6df4eefd5b1a4823c96e574a19
+make_check=ci-skip # cyclic dependency
```

Comment:
It's not self-depencency, but need for files that are created during install phase rather than build. A bug that can be reported upstream. Please mark `make_check=no # doesn't create egg-info during build` instead.